src/GraphQl/Queries/Queries.ts

```diff
@@ -254,15 +230,8 @@ export const MEMBERSHIP_REQUEST = gql`
 // display posts
 
 export const ORGANIZATION_POST_LIST = gql`
-  query PostsByOrganization(
-    $id: ID!
-    $filterByTitle: String
-    $filterByText: String
-  ) {
-    postsByOrganization(
-      id: $id
-      where: { title_contains: $filterByTitle, text_contains: $filterByText }
-    ) {
```

Comment:
Why we need to remove the filters ?

Reply:
refer this
https://thepalisadoes-dyb6419.slack.com/archives/C01RPC00K0D/p1674557631837709

Reply:
![image](https://user-images.githubusercontent.com/67755128/216835677-cb433533-0b2f-43ea-bdda-de1525f1fa72.png)
I do not think we require any filter option as well, because we can get the info by id

if we need a filter then use this
```
    postsByOrganizationConnection(
      id: ID!
      where: PostWhereInput
      first: Int
      skip: Int
      orderBy: PostOrderByInput
    ): PostConnection
```

Reply:
okay, got the point.

Reply:
@noman2002 @palisadoes @VenomFate-619 I have a doubt here.
- In `organizationsConnection.ts`, we are returning `[Organization]`. This implies we have no option for pagination here.
- In `postsByOrganizationConnection.ts`, we are returning `PostConnection`, which is an object of the form
```javascript
{
      pageInfo: {
        hasNextPage: boolean,
        hasPreviousPage: boolean,
        totalPages: integer,
        nextPageNo: integer,
        prevPageNo: integer,
        currPageNo: integer,
      },
      edges: posts,
      aggregate: {
        count: integer,
      },
}
```
Here, we have an option for pagination.
So, should we replace `PostConnection` with `[Post]` in `postsByOrganizationConnection.ts` and the `resolver` accordingly because we are nowhere using pagination using `talwa-api` in our entire codebase? With this we will be able to get filtered data, like we did in the case of filtering Organizations based on `name_contains`.
```
  query OrganizationsConnection($filter: String) {
    organizationsConnection(where: { name_contains: $filter }) {
      _id
      image
      creator {
        firstName
        lastName
      }
      name
      members {
        _id
      }
      admins {
        _id
      }
      createdAt
      location
    }
  }
```
Should we create an issue for this in `talawa-api`?